resources/js/Components/AuthWidget.vue

```diff
@@ -1,26 +1,22 @@
 <template>
     <div class="auth-widget" v-if="user">
-        <WikitLink :href="`https://www.wikidata.org/wiki/User:${user.name}`">
+        <a :href="`https://www.wikidata.org/wiki/User:${user.name}`">
             <img src="images/user.svg" class="icon-user" /><span class="username">{{ user.name }}</span>
-        </WikitLink>
-        <WikitLink href="/auth/logout">{{ $i18n('log-out') }}</WikitLink>
+        </a>
+        <a  href="/auth/logout">{{ $i18n('log-out') }}</a>
     </div>
-    <WikitLink v-else href="/auth/login">{{ $i18n('log-in') }}</WikitLink>
+    <a v-else href="/auth/login">{{ $i18n('log-in') }}</a>
```

Comment:
Now there's a focus border around the link that wasn't there 

![image](https://github.com/wmde/wikidata-mismatch-finder/assets/3005239/31b2e456-0090-43bc-b26a-61f405290021)

The focus border is also there around the username and the log out links. 

![image](https://github.com/wmde/wikidata-mismatch-finder/assets/3005239/711df771-d14f-4f71-9e93-8521fe38aa4e)

Reply:
This is built into codex link. But since we host the mixin locally we could theoretically get rid of it. What does @sai-san think?

Reply:
This is an accessibility improvement to make clear that links are receiving focus. We shouldn't get rid of this style.
I'm wondering about that apparent extra space around the username link, though. I could take a deeper look at what's causing that in staging.

Reply:
@sai-san and I discussed this offline and ended up adjusting the styles for the auth widget to fix the above issue

Reply:
Looks great now. Thx